e2e/e2e_client_server.py

```diff
@@ -0,0 +1,41 @@
+
+import argparse
+import utils.CallBacks
+from utils.BluetoothHandler import BluetoothHandler
+
+def main():
+    parser = argparse.ArgumentParser()
+    group = parser.add_mutually_exclusive_group(required=True)
+    group.add_argument("-i", help="flag indicating if should initiate as a server",
+                    action="store_true")
+    group.add_argument("--serverMac", help='MAC address of the server for bluetooth connection', type=str)
+
+    parser.add_argument('--nativeLanguage', choices=['en', 'ch', 'terminal'], required=True)
+    parser.add_argument('-v', '--verbose', default=False)
+    args = parser.parse_args()
+
+    # Create callbacks based on language
+    if args.nativeLanguage == 'en':
+        sendingCallback = utils.CallBacks.getInputFromEnglish
+        receivingCallback = [utils.CallBacks.outputInputEnglish]
+    elif args.nativeLanguage == 'ch':
```

Comment:
```suggestion
    elif args.nativeLanguage == "zh":
```

I think the ISO 639 code for Chinese is zh!